vagrant/provision.sh

```diff
@@ -88,7 +88,7 @@ curl -Lo- "https://github.com/koalaman/shellcheck/releases/download/${SHELLCHECK
 sudo cp "shellcheck-${SHELLCHECK_VERSION}/shellcheck" /usr/local/bin
 rm -rf "shellcheck-${SHELLCHECK_VERSION}/"
 
-gem install mdl
+npm install -g markdownlint-cli
```

Comment:
npm is missing on Vagrant VM

Reply:
😱

Reply:
npm can be installed using `apt-get install npm`

Reply:
let me fix that

Reply:
Done